dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaYarnApplicationTypeTask.java

```diff
@@ -72,21 +74,28 @@ protected void doRun(
         new PaginatedClouderaYarnApplicationsLoader(
             handle, context.getArguments().getPaginationPageSize());
 
+    List<ClouderaYarnApplicationDTO> sparkYarnApplications = new ArrayList<>();
     try (Writer writer = sink.asCharSink(StandardCharsets.UTF_8).openBufferedStream()) {
       for (ClouderaClusterDTO cluster : clusters) {
         final String clusterName = cluster.getName();
-        Set<String> yarnAppTypes = new HashSet<>(fetchYARNApplicationTypes(handle, clusterName));
-        yarnAppTypes.addAll(predefinedAppTypes);
```

Comment:
I think it was a reason to have this `predefinedAppTypes`. @zaldis  Do you remember?